packages/common/src/index.ts

```diff
@@ -481,7 +486,9 @@ type ApiResults =
     | PinnedItems
     | ViewStatistics
     | SchedulerWithLogs
-    | ValidationResponse[];
+    | ValidationResponse[]
+    | ApiJobStatusResponse['results']
```

Comment:
By using `['results']` we can reuse the same types created for the controller response, instead of duplicating types. But let me know if you don't like it